RevenueCatUI/CustomerCenter/ViewModels/CustomerCenterViewModel.swift

```diff
@@ -99,6 +102,11 @@ import RevenueCat
         self.customerCenterStoreKitUtilities = customerCenterStoreKitUtilities
     }
 
+    convenience init(uiPreviewPurchaseProvider: CustomerCenterPurchasesType) {
+        self.init(actionWrapper: CustomerCenterActionWrapper(legacyActionHandler: nil),
+                  purchasesProvider: uiPreviewPurchaseProvider)
+    }
```

Comment:
This `init` is used by the new `@_spi(Internal) public init` of `CustomerCenterView` that receives a parameter `uiPreviewPurchaseProvider: CustomerCenterPurchasesType`